Makefile.build_config.in

```diff
@@ -43,3 +43,6 @@ DOCUMENTATION_TOOL_CMD=@documentation_tool_cmd@
 # Git submodule)
 FLEXDLL_SOURCES=@flexdir@
 BOOTSTRAPPING_FLEXDLL=@bootstrapping_flexdll@
+
+### Where to install documentation
+DOCDIR=@docdir@
```

Comment:
```suggestion
PACKAGE_TARNAME = @PACKAGE_TARNAME@
datarootdir = @datarootdir@
DOCDIR=@docdir@
```
Defining `datarootdir` is necessary to avoid this:
```
config.status: creating Makefile.build_config
config.status: WARNING:  'Makefile.build_config.in' seems to ignore the --datarootdir setting
config.status: creating Makefile.config
config.status: creating stdlib/sys.ml
```
CI has picked up the missing `PACKAGE_TARNAME` definition. autoconf wants to `DOCDIR=${prefix}/share/doc/${PACKAGE_TARNAME}` by default.

Reply:
Thanks David, should have checked before committing!